app/(client)/faq/page.tsx

```diff
@@ -1,9 +1,141 @@
+
 import { FAQAccordion } from '@/components/ui/accordion';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbList,
+  BreadcrumbSeparator,
+} from '@/components/ui/breadcrumb';
+import { Button } from '@/components/ui/button';
 
-export default async function FAQ() {
+export default function FAQ() {
   return (
     <>
-      <FAQAccordion />
+      <header className="w-full flex flex-col justify-around">
+        <Breadcrumbs />
+      </header>
+      <main className="flex xsm:flex-col sm:flex-col md:flex-col lg:flex-row min-[390px]:flex-col max-[390px]:flex-col p-1 gap-2">
+        <AsideCategories />
+        <FAQAccordion />
+      </main>
     </>
   );
 }
+
+const Breadcrumbs = () => {
+  return (
+    <Breadcrumb className="h-20 flex flex-row mx-3">
+      <BreadcrumbList>
+        <BreadcrumbItem>
+          <Layers /> Головна
+        </BreadcrumbItem>
+        <BreadcrumbSeparator />
+        <BreadcrumbItem>
+          <BreadcrumbLink className="flex flex-row xsm:items-center gap-1">
+            <Cable /> Часті питання
+          </BreadcrumbLink>
+        </BreadcrumbItem>
+      </BreadcrumbList>
+    </Breadcrumb>
+  );
+};
+
+function ToggleAccordionItems(value: string) {
+  const ids: string[] = [];
+
+  // For which button value which accordion items with according ids should be displayed. If id is not present
+  // in the dictionary - hide.
+  const toggle_ids: { [value: string]: string[] } = {
+    general: ['how-to-join-clubs', 'how-to-be-in-touch'],
+    documents: [
+      'where-to-find-required-document',
+      'what-documents-are-required',
+    ],
+    info: ['cannot-find-info'],
+    studrad: ['how-to-join-clubs'],
+    students: [
+      'conflict-with-professeur',
+      'cannot-find-info',
+      'how-to-be-in-touch',
+    ],
+    common: [
+      'what-documents-are-required',
+      'how-to-be-in-touch',
+      'how-to-join-clubs',
+      'cannot-find-info',
+      'do-not-understand',
+      'conflict-with-professeur',
+      'where-to-find-required-document',
+    ],
+    admission: [],
+    org: [],
+  };
+
+  if (value in toggle_ids) {
+    if (document) {
+      const children = document.getElementById('faq-accordion')!.childNodes;
+
+      for (let index = 0; index < children.length; index++) {
+        const child = children[index] as HTMLElement;
+        ids.push(child.id);
+
+        child.classList.remove('hidden');
+        if (!toggle_ids[value].includes(child.id)) {
+          child.classList.add('hidden');
+        }
+      }
+    }
+  }
+}
+
+const ToggleButton = ({ text, value }: { text: string; value: string }) => {
+  return (
+    <Button
+      variant={'outline'}
+      size={'sm'}
+      className="w-full not-italic font-normal leading-normal text-base sm:text-xl xsm:text-xl"
+      value={value}
+      onClick={event => {
+        ToggleAccordionItems(event.currentTarget.value);
+      }}
+    >
+      {text}
+    </Button>
+  );
+};
+
```

Comment:
It is a redundant button component, you can write styles for button and map with an array of values from the backend